include/mockturtle/views/fanout_view.hpp

```diff
@@ -111,6 +116,21 @@ class fanout_view<Ntk, false> : public Ntk
     _fanout[ n ].push_back( p );
   }
 
+  void substitute_node_of_parents( std::vector<node> parents, node const& old_node, signal const& new_signal )
+  {
+    this->substitute_node_of_parents( parents, old_node, new_signal );
```

Comment:
Should be `Ntk::substitude_node_of_parents` to call the original implementation.